test/interaction/interaction.test.js

```diff
@@ -305,6 +306,32 @@ describe('resume after interaction', () => {
     });
   });
 
+  context('meta results', () => {
+    it('should process and store meta-informations provided alongside login', function () {
+      const auth = new this.AuthorizationRequest({
+        response_type: 'code',
+        scope: 'openid',
+      });
+
+      setup.call(this, auth, {
+        login: {
+          account: uuid(),
+          remember: true,
+        },
+        meta: {
+          scope: 'openid',
+        },
+      });
+
+      return this.agent.get('/auth/resume')
+        .expect(() => {
+          const authorization = this.getSession().authorizations[config.client.client_id];
```

Comment:
can you use metaFor here? This would cover the last line for codecov

Reply:
it seems that the `TestAdapter` doesn't return models' instances but raw objects; 

> this.getSession(...).metaFor is not a function

Any ideas ?

Reply:
no ideas, just test the metaFor function then.